src/setupEnzyme.tsx

```diff
@@ -39,6 +39,10 @@ export function setupEnzyme<
     } as FullProps<ComponentType>;
     const wrapper = mount(<Component {...(props as any)} />);
 
-    return { props, wrapper };
+    // setProps demands _something_ be passed, so we keep that going, too
+    const update = (updatedProps: Pick<FullProps<ComponentType>, keyof FullProps<ComponentType>>) =>
```

Comment:
Applying the `Pick` suggestion here:

```suggestion
    const update = (updatedProps: Partial<FullProps<ComponentType>>) =>
```

Reply:
I'd tried that, but it seems TypeScript doesn't realize that these are effectively the same =(

![Screen Shot 2021-04-28 at 11 44 15 AM](https://user-images.githubusercontent.com/13091526/116432975-1120ab80-a817-11eb-9d4c-f8bb837ebf0d.png)

Reply:
Hmm, where's that `any` coming from? Either this is a TypeScript bug or there's an underlying type oddity 🤔 

_(leaning towards the latter by default)_

Not a blocker from me either way!

Reply:
Yeah, it's a disconnect within the `enzyme` types for this method =(